### PR TITLE
Add language selector

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Needs: https://github.com/home-assistant/core/pull/91669

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
Let's also update `pick-language-row`

File: src/components/ha-language-picker.ts [hunk 1]
```diff
@@ -30,27 +30,40 @@ export class HaLanguagePicker extends LitElement {
...
-      const options = supportedLanguages.map((lang) => ({
-        value: lang,
-        label: languageDisplayNames ? languageDisplayNames.of(lang)! : lang,
-      }));
+    (languages: string[], language: string, nativeName: boolean) => {
+      let options: { label: string; value: string }[] = [];
+
+      if (nativeName) {
+        const translations = this.hass.translationMetadata.translations;
+        options = languages.map((lang) => ({
```
Comment (bramkragten) [inline src/components/ha-language-picker.ts:48]: Should we sort the options?

**Review by bramkragten — APPROVED** [review]

**Comment by bramkragten** [comment]
Backend is not merged?

**Comment by piitaya** [comment]
Oops, I will revert it

**Pull request merged**